docs/core/howto/defer-intro.rst

```diff
@@ -401,8 +401,10 @@ Coroutines with async/await
 
     .. versionadded:: 16.4
 
-On Python 3.5 and higher, the :pep:`492` ("Coroutines with async and await syntax") "await" functionality can be used with Deferreds by the use of :api:`twisted.internet.defer.ensureDeferred <ensureDeferred>`.
+On Python 3.5 and higher, the :pep:`492` ("Coroutines with async and await syntax") "await" functionality can be used with Deferreds by the use of :api:`twisted.internet.defer.coroFnToDeferredFn <coroFnToDeferredFn>`.
+
 It is similar to ``inlineCallbacks``, except that it uses the ``await`` keyword instead of ``yield``, the ``return`` keyword instead of ``returnValue``, and is a function rather than a decorator.
```

Comment:
```suggestion
It is similar to ``inlineCallbacks``, except that it uses the ``await`` keyword instead of ``yield``, the ``return`` keyword instead of ``returnValue``.
```

Reply:
I have never used Twisted with py3 and I have never used ensureDeferred... so maybe for someone else this is obvious.

But to me it is not clear why ensureDeferred was removed from this documentation and why we need the new `coroFnToDeferredFn`